types/ember-cli-fastboot/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
```

Comment:
We actually just want this to be `"strict": true`! This matches what other Ember packages have… until later today when I open the MAMMOTH update for v4 which will include that update.

Reply:
done!